utilities/src/Types/Eventprocessor.res

```diff
@@ -0,0 +1,4 @@
+type eventProcessor<'a, 'b, 'c> = {
+  processor: (~event: Event.event<'b>, ~hint: Event.eventHint<'c>) => 'a,
+  id?: string, // This field can't be named "name" because functions already have this field natively
```

Comment:
if sentry has key as name then we cant change it

Reply:
Yeah we can't